helm-charts/secrets-operator/values.yaml

```diff
@@ -32,7 +32,7 @@ controllerManager:
         - ALL
     image:
       repository: infisical/kubernetes-operator
-      tag: v0.5.0 # fixed to prevent accidental upgrade
+      tag: v0.5.1 # fixed to prevent accidental upgrade (manually bump when adding new versions)
```

Comment:
just a note: for breaking changes, we don't change this because then it will auto pull the latest one and break thing current setup